modules/ROOT/pages/migration-from-6x.adoc

```diff
@@ -24,41 +26,257 @@ Open Source users: please report issues in the https://github.com/tinymce/tinymc
 * xref:ui-and-ux-elements-and-components[UI and UX elements and components]
 * xref:security[Security]
 
+[[tinymce-70-core-changes]]
 == {productname} {productmajorversion} core changes
 
 For additional details on {productname} 7.0 changes, see xref:7.0-release-notes.adoc[{productname} 7.0 release notes].
 
+
+[[breaking-changes]]
+== Breaking Changes
+
+TODO//
+
 [[apis]]
 == APIs
 
-...
+TODO//
 
 [[commands-and-queries]]
 == Commands and queries
 
-...
+TODO//
 
 [[options]]
 == Options
 
-...
+[[sandbox-iframes-option]]
+=== `sandbox_iframes`
+
+Previous Security penetration testing surfaced a security issue with {productname} loading iframes `unsandboxed` which asa. result could allow arbitrary code execution inside {productname} content.
```

Comment:
Did some of the text get removed `with asa.` ?

Reply:
```suggestion
Previous Security penetration testing surfaced a security issue with {productname} loading iframes `unsandboxed` which as a result could allow arbitrary code execution inside {productname} content.
```